### PR TITLE
Update connect-to-gcc-data-with-usage-analytics.md

### LINKED ISSUE #12435: Needs a note to distinguish GCC from GCCH as this is not supported in the latter.
Hi Team - this is not currently a supported feature in GCCH, the article is a bit ambiguous and it would be helpful to call out that this is for GCCModerate, and not GCCH/DOD.  Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6be7caf9-523d-2e84-fb17-14b1606aa97a
* Version Independent ID: 34a7cb37-5f79-c1d3-f40f-8afb91f48862
* Content: [Connect to Microsoft 365 Government Community Cloud (GCC) data with Usage Analytics - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/usage-analytics/connect-to-gcc-data-with-usage-analytics?view=o365-worldwide)
* Content Source: [microsoft-365/admin/usage-analytics/connect-to-gcc-data-with-usage-analytics.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/usage-analytics/connect-to-gcc-data-with-usage-analytics.md)
* Service: **microsoft-365-business**
* GitHub Login: @efrene
* Microsoft Alias: **efrene**

### PR DESCRIPTION
**Pull request by msbemba**
Added a statement to reduce ambiguity on GCCH

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12435

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/usage-analytics/connect-to-gcc-data-with-usage-analytics.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ description: "Learn how to connect to data in your Microsoft 365 Government Comm
 Use the following procedures to connect to your data with the Microsoft 365 Usage Analytics report in a Microsoft 365 Government Community Cloud (GCC) tenant. 
 
 > [!NOTE]
-> These instructions are specifically for Microsoft 365 GCC tenants. 
+> These instructions are specifically for Microsoft 365 GCC tenants and not applicate for not GCC HIgh/DOD.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/connect-to-gcc-data-with-usage-analytics.md]:
```suggestion
> These instructions are specifically for Microsoft 365 GCC tenants and are not applicate to GCC HIgh and DOD.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Pull request merged**